postprocessing.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
```

Comment:
Please add a description. Otherwise it's not clear what the script is used to.

Reply:
Done !